modules/noise/doc_classes/FastNoiseLite.xml

```diff
@@ -0,0 +1,208 @@
+			Octave gain for all fractal noise types.
+		</member>
+		<member name="fractal_lacunarity" type="float" setter="set_fractal_lacunarity" getter="get_fractal_lacunarity" default="2.0">
+			Octave lacunarity for all fractal noise types.
+		</member>
+		<member name="fractal_octaves" type="int" setter="set_fractal_octaves" getter="get_fractal_octaves" default="5">
+			The number of noise layers that are sampled to get the final value for fractal noise types.
+		</member>
+		<member name="fractal_ping_pong_strength" type="float" setter="set_fractal_ping_pong_strength" getter="get_fractal_ping_pong_strength" default="2.0">
+			Sets the strength of the fractal ping pong type.
+		</member>
+		<member name="fractal_type" type="int" setter="set_fractal_type" getter="get_fractal_type" enum="FastNoiseLite.FractalType" default="1">
+			Change the type of fractal mode. See [enum FractalType].
+		</member>
+		<member name="fractal_weighted_strength" type="float" setter="set_fractal_weighted_strength" getter="get_fractal_weighted_strength" default="0.0">
+			Higher weighting means higher octaves have less impact if lower octaves have a large impact
+		</member>
+		<member name="frequency" type="float" setter="set_frequency" getter="get_frequency" default="0.01">
+			The frequency for all noise types.
+		</member>
+		<member name="in_3d_space" type="bool" setter="set_in_3d_space" getter="is_in_3d_space" default="false">
+			Determines whether the noise image returned by [method get_image] is calculated in 3d space. May result in reduced contrast.
+		</member>
+		<member name="noise_type" type="int" setter="set_noise_type" getter="get_noise_type" enum="FastNoiseLite.NoiseType" default="1">
+			The noise algorithm used. See [enum NoiseType].
+		</member>
+		<member name="offset" type="Vector3" setter="set_offset" getter="get_offset" default="Vector3(0, 0, 0)">
+			Translate the noise input coordinates by the given [Vector3].
+		</member>
+		<member name="seed" type="int" setter="set_seed" getter="get_seed" default="0">
+			The random number seed for all noise types.
+		</member>
+	</members>
+	<constants>
+		<constant name="TYPE_VALUE" value="5" enum="NoiseType">
+			A lattice of points are assigned random values then interpolated based on neighboring values.
+		</constant>
+		<constant name="TYPE_VALUE_CUBIC" value="4" enum="NoiseType">
+			Similar to Value noise, but slower. Has more variance in peaks and valleys.
+		</constant>
+		<constant name="TYPE_PERLIN" value="3" enum="NoiseType">
+			A lattice of random gradients. Their dot products are interpolated to obtain values in between the lattices.
+		</constant>
+		<constant name="TYPE_CELLULAR" value="2" enum="NoiseType">
+			Cellular includes both Worley noise and Voronoi diagrams which creates various regions of the same value.
+		</constant>
+		<constant name="TYPE_SIMPLEX" value="0" enum="NoiseType">
```

Comment:
An important distinguishing feature of simplex noise over other gradient noise types is that the gradients exist in a simplex lattice rather than a grid lattice. This helps avoid directional artifacts that perlin noise suffers from. It also means that it scales better with an increase in dimensions (once you hit 4 or more dimensions simplex noise becomes much faster than perlin)